lib/issues2eslint.js

```diff
@@ -17,8 +17,31 @@ const mythx2Severity = {
     Medium: 1,
 };
 
+const logLevel2Severity = {
```

Comment:
Super minor and nitpicky suggestion: This variable name is slightly confusing because "log level" could also refer to logs that MythX returns instead of issues. Perhaps `severityToNumber` is a better name?

Reply:
that's a clearer name, agreed! since updated.